server/src/main/scala/uk/gov/homeoffice/drt/crunchsystem/ReadRouteUpdateActorsLike.scala

```diff
@@ -0,0 +1,19 @@
+package uk.gov.homeoffice.drt.crunchsystem
+
+import actors.MinuteLookupsLike
+import akka.actor.ActorRef
+
+trait ReadRouteUpdateActorsLike {
```

Comment:
I'm not sure what to call this really as it's a collection of actors that don't really have anything in common with each other. 

Maybe `ActorsServiceLike` given than it's implemented by `ActorsService`?